### PR TITLE
Using jackson and jackson_databind version defined in OpenSearch

### PR DESCRIPTION
**Pull request by penghuo**
Signed-off-by: Peng Huo <penghuo@gmail.com>

### Description
Using jackson and jackson_databind version defined in OpenSearch.  SQL plugins's jackson depedency will automatically allign with OpenSearch core then.

 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by dai-chen — APPROVED** [review]
Thanks for the changes!

File: build.gradle [hunk 1]
```diff
@@ -69,6 +67,14 @@ plugins {
     id 'jacoco'
 }
 
+// import versions defined in https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/src/main/java/org/opensearch/gradle/OpenSearchJavaPlugin.java#L94
+apply plugin: 'opensearch.java'
+// versions https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/version.properties
+ext {
+    jackson_version = versions.jackson
```
Comment (YANG-DB) [inline build.gradle:74]: Shouldn’t we conform to the same version name convention ?

**Review by YANG-DB — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (penghuo) [reply build.gradle:74]: I did not get it. do you mean using version.jackson directly?

**Review by joshuali925 — DISMISSED** [review]

File: build.gradle (hunk 1 above)
Comment (YANG-DB) [reply build.gradle:74]: yes 👍

**Review by joshuali925 — APPROVED** [review]

**Review by YANG-DB — APPROVED** [review]

**Pull request merged**